src/lib/fs/rename.js

```diff
@@ -0,0 +1,12 @@
+import { rename } from 'node:fs/promises';
+import path from 'node:path';
+
+export default async function renameFn(pathToFile, newFileName) {
+  try {
+    const resolvedPath = path.resolve(pathToFile);
+
+    await rename(resolvedPath, newFileName);
+  } catch (err) {
+    throw err;
```

Comment:
one throw works the same as without catch block. The code throws an error that should be handled higher